integration-test/trace-recorder/src/main/org/jetbrains/trace/recorder/test/impl/generated/KotlinCompilerTests.kt

```diff
@@ -0,0 +1,493 @@
+@file:Suppress("ClassName")
+
+package org.jetbrains.trace.recorder.test.impl.generated
+
+import org.jetbrains.trace.recorder.test.impl.*
+import org.jetbrains.trace.recorder.test.runner.*
+import org.junit.Test
+import org.junit.experimental.categories.Category
+
+/*
+  THIS FILE IS AUTOGENERATED!
+  DO NOT MODIFY!
+*/
+
+@Category(KotlinCompilerTraceRecorderTest::class)
+class KotlinCompilerTestsTraceRecorderIntegrationTest_FirLightTreeBlackBoxCodegenTestGenerated_testAllFilesPresentInBox : KotlinCompilerTraceRecorderJsonTests() {
```

Comment:
Why do we have separation by classes instead of one class with several methods? Is this TC restriction?